### PR TITLE
Storage timestamp ng

### PR DESCRIPTION
**Pull request by jreidinger**
intentionally do not increase version.

### CODE REVIEW

File: test/boot_storage_test.rb [hunk 1]
```diff
@@ -89,6 +89,18 @@
 
       expect(subject.mbr_disk).to eq "/dev/vda"
     end
+
+    it "skips cache if storage gets changed" do
+      subject.RootPartitionDevice = "/dev/sda1"
+      subject.BootPartitionDevice = "/dev/sda2"
+      subject.mbr_disk = "/dev/sda"
+
+      allow(Yast::Storage).to receive(:GetTargetChangeTime).and_return(Time.now.to_i)
```
Comment (ancorgs) [inline test/boot_storage_test.rb]: Hmmm

File: test/boot_storage_test.rb [hunk 2]
```diff
@@ -89,6 +89,18 @@
...
+
+      subject.detect_disks
+
+      expect(subject.RootPartitionDevice).to eq "/dev/vda1"
+    end
```
Comment (ancorgs) [inline test/boot_storage_test.rb]: Now that I saw the previous test passing even it was wrong, I'm kind of worried. Can we have a complementary test or something?

Comment (jreidinger) [inline test/boot_storage_test.rb]: see xdescribe on top of file. This test is disabled for now. As its testing require solid set of device graph mocking, which is currently a bit out of scope of this PR.

Comment (ancorgs) [inline test/boot_storage_test.rb]: Ah, ok

**Comment by jreidinger** [comment]
@ancorgs thanks

**Pull request merged**